### PR TITLE
Working on #2972 Template Conversion to Twig Format (private.php)

### PR DESCRIPTION
**Pull request by PaulBender**
#2972 for the private message system (private.php).

### CODE REVIEW

**Comment by PaulBender** [comment]
This is finished.

**Comment by Ben-MyBB** [comment]
Great work!

File: inc/views/base/private/export/html.twig [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		<style type="text/css">{{ css|raw }}
+		* {text-align: left}</style>
+	</head>
+	<body>
+		<br />
+		<table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+			<tr>
+				<td class="thead" colspan="1"><span class="largetext"><strong>{{ lang.private_messages_for }}</strong></span></td>
+			</tr>
+            {% for message in pmsdownload %}
```
Comment (euantorano) [inline inc/views/base/private/export/html.twig]: Indentation here looks a bit off.

Comment (PaulBender) [reply inc/views/base/private/export/html.twig]: Fixed

**Pull request merged**